File: HEUR-Trojan-Ransom.MSIL.Thanos.gen-66ed5384220ff3091903e14a54849f824fdd13ac70dc4e0127eb59c1de801fc2/frgoWklvEOSV/AnLYutpDjmCX.cs
using System;

namespace frgoWklvEOSV;

public sealed class AnLYutpDjmCX
{
	public static string cVQEpGjgoi(int P_0)
	{
		Random random = new Random();
		string text = null;
		for (int i = 0; i < 32; i++)
		{
			char value = (char)(random.Next(33, 127) % 255);
			if (Convert.ToInt32(value) == 34 || Convert.ToInt32(value) == 92)
			{
				i--;
			}
			else
			{
				text += Convert.ToChar(value);
			}
		}
		return text;
	}
}

File: HEUR-Trojan-Ransom/MSIL/T/HEUR-Trojan-Ransom.MSIL.Thanos.gen-d29abe6ed086a5508c54df31010c36cc19fea3bdc5d521ee7c0d7063a51bb131/chaddad/frgoWklvEOSV/AnLYutpDjmCX.vb
Imports System

Namespace frgoWklvEOSV
	' Token: 0x0200000C RID: 12
	Public NotInheritable Class AnLYutpDjmCX
		' Token: 0x0600003D RID: 61 RVA: 0x0000BEF4 File Offset: 0x0000A0F4
		Public Shared Function cVQEpGjgoi(int_0 As Integer) As String
			Dim random As Random = New Random()
			Dim text As String = Nothing
			Dim i As Integer = 0
			While i < 32
				Dim c As Char = CChar((random.[Next](33, 127) Mod 255))
				If Convert.ToInt32(c) = 34 Then
					GoTo IL_48
				End If
				If Convert.ToInt32(c) = 92 Then
					GoTo IL_48
				End If
				text += Convert.ToChar(c)
				IL_4C:
				i += 1
				Continue While
				IL_48:
				i -= 1
				GoTo IL_4C
			End While
			Return text
		End Function
	End Class
End Namespace
